cuda.spec

```diff
@@ -101,6 +102,7 @@ ln -sf libnvidia-ptxjitcompiler.so.1                                        %{i}
 sed \
   -e"/^TOP *=/s|= .*|= $CMS_INSTALL_PREFIX/%{pkgrel}|" \
   -e's|$(_HERE_)|$(TOP)/bin|g' \
+  -e's|$(TOP)/lib|$(TOP)/lib64|g' \
```

Comment:
this looks correct, but in fact should not be needed, after scram has set up the environment ?

Reply:
Indeed it's not really needed. I reverted all the changes on cuda.spec now.